src/java/ognl/AccessibleObjectHandler.java

```diff
@@ -0,0 +1,51 @@
+//--------------------------------------------------------------------------
+//	Copyright (c) 1998-2004, Drew Davidson and Luke Blanshard
```

Comment:
I would either put your credits or use a standard ASL 2.0 header instead - this code cannot be credited by the above authors :)

Reply:
I wasn't sure of the best header to use (given the differences in the source and license.txt file),  
I've attempted to use a slight modification of the ASF header from some Struts files that seemed to fit the circumstances reasonably well.  
All three files have been updated with that header - let me know if it looks OK now,